qemu/tests/cfg/hotplug_mem_share_discard_data.cfg

```diff
@@ -0,0 +1,35 @@
+- hotplug_mem_share_discard_data:
+    only Linux
```

Comment:
Are there test cases for both qemu6.2 and qemu ?

Reply:
Yes